### DISCUSSION TITLE
How can I highlight ft.ListTile after clicking?

### DISCUSSION THREAD

**Discussion by dvd1221** [flet-dev/flet#1828] [Q&A]
I want to create a list of selectable items. When I click, information about the selected item appears. I want the selected option to be highlighted until I select another one.

![image](https://user-images.githubusercontent.com/126869058/225645636-9a6c8374-81c6-479d-bcc1-38cba46b726d.png)

```
dict_py_proj = {'Расчет на разрыв':'C:\PyProjects\pythonProject_flet_stability\main.py',
                'Расчет на смятие':'C:\PyProjects\python_project_flet\main.py'}
page.clean()
title = ft.Text("Заголовок", style=ft.TextThemeStyle.TITLE_LARGE)
title_row = ft.Row([title], alignment=ft.MainAxisAlignment.CENTER)

col = ft.Column([])
for key in dict_py_proj.keys():
    col.controls.append(
                ft.ListTile(
                    title = ft.Text(key),
                    selected=True,
                    autofocus=False,
                    dense=True,
                    on_click=show_doc
    ))

page.add(title_row, col)
page.update()
```

**Comment by mikaelho** [comment]
You have created all ListTiles as selected=True. Remove that and instead:
- Maintain a variable that holds the currently_selected ListTile or None.
- In show_doc event handler, set currently selected ListTile selected=False and the new one to True (and store the new one in the variable).
- Don't forget to update().

**Comment by dvd1221** [comment]
@mikaelho , Which method can I use to determine the selected item? I don't quite understand.. How to access the selected element?
